src/rpc/RPCHelpers.cpp

```diff
@@ -1674,13 +1680,9 @@ traverseTransactions(
             obj[JS(date)] = txnPlusMeta.date;
         }
         obj[JS(validated)] = true;
-
         txns.push_back(obj);
     }
 
-    response[JS(ledger_index_min)] = minIndex;
-    response[JS(ledger_index_max)] = maxIndex;
```

Comment:
Why did we remove this? Sorry I didn't flag this before.

Reply:
@cjcobb23 mistakenly removed when accounting for changes regarding deletions for maxReturnedIndex and minReturnedIndex. Restored in latest push, successful result
![image](https://user-images.githubusercontent.com/47958858/192046871-c09d864a-7415-4454-ace9-2cd2fa59d4a0.png)